packages/amplify-category-function/src/provider-utils/awscloudformation/utils/consolidateDependsOn.ts

```diff
@@ -0,0 +1,30 @@
+import { $TSMeta } from 'amplify-cli-core';
+import { FunctionDependency } from 'amplify-function-plugin-interface';
+import _ from 'lodash';
```

Comment:
It looks like `get()` is the only lodash method used here. Can we use optional chaining and nullish coalescing instead?

Reply:
replaced. Is there a disadvantage using lodash?